src/webvr-manager.js

```diff
@@ -98,9 +98,10 @@ function WebVRManager(renderer, effect, params) {
       this.isVRCompatible = true;
     } else if (hmd) {
       this.isVRCompatible = true;
+      this.usingPolyfill = hmd.deviceName.indexOf('webvr-polyfill') === 0;
       // Only enable distortion if we are dealing using the polyfill, we have a
```

Comment:
Could you check this into the webvr-1.0 branch? That's where the action is
happening.

On Sat, Mar 26, 2016 at 12:27 PM Brian Peiris <notifications@github.com>
wrote:

> In src/webvr-manager.js
> <https://github.com/borismus/webvr-boilerplate/pull/113#discussion_r57518253>
> :
>
> > @@ -98,9 +98,10 @@ function WebVRManager(renderer, effect, params) {
> >        this.isVRCompatible = true;
> >      } else if (hmd) {
> >        this.isVRCompatible = true;
> > +      this.usingPolyfill = hmd.deviceName.indexOf('webvr-polyfill') === 0;
> >        // Only enable distortion if we are dealing using the polyfill, we have a
>
> Oh, looks like that hasn't been released. Se maybe we can merge this now
> and update it when polyfill is updated.
>
> —
> You are receiving this because you commented.
>
> Reply to this email directly or view it on GitHub
> <https://github.com/borismus/webvr-boilerplate/pull/113/files/e2f8f70aa86f04e134227665557b3884618a061c#r57518253>
>